src/integrationTest/java/uk/gov/hmcts/reform/cdapi/RetrieveCategoriesIntegrationTest.java

```diff
@@ -30,6 +33,56 @@ public class RetrieveCategoriesIntegrationTest extends CdAuthorizationEnabledInt
 
     private static final String path = "/lov/categories/{category-id}";
 
+    @Test
+    @DisplayName("Retrieve categories with externalRefarence ")
```

Comment:
Typo, "externalRefarence" should be "externalReference"

Reply:
corrected